DragonSlayer.cpp

```diff
@@ -1,20 +1,38 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
+#include "Utility.h"
+#include "AttackItem.h"
 
-//DragonSlayer::DragonSlayer
-
-//DragonSlayer::getName
+DragonSlayer::DragonSlayer(const std::string& name_, int hp_, int armor_) : Character(hp_, armor_, 4), name(name_)
+{
+    helpfulItems = makeHelpfulItems(1);
+    defensiveItems = makeDefensiveItems(1);
+    attackItems = makeAttackItems(4);
```

Comment:
Follow the spec.
Dragon Slayers get ONE attack item, not a vector of them
```
 7) create a new Item-derived type that boosts the attack damage called AttackItem 
        add an instance of it to the DragonSlayer class
        make the DragonSlayer::attack use it when the dragonSlayer attacks a dragon. 

```
- [ ] revise